test/unit/utils/test_utils.cpp

```diff
@@ -50,12 +50,13 @@ NEURON {
  * nmodl, the nmodl output is without "extra" whitespaces in the provided input.
  */
 
-std::string reindent_text(const std::string& text) {
+std::string reindent_text(const std::string& text, int indent_level) {
```

Comment:
I take it this (and below) is part of https://github.com/BlueBrain/nmodl/pull/1549?

Reply:
correct.